lstchain/scripts/onsite/onsite_create_calibration_files_with_batch.py

```diff
@@ -0,0 +1,155 @@
+import lstchain
+import subprocess
+import shutil
+
+# parse arguments
+parser = argparse.ArgumentParser(description='Reconstruct filter scan, this must be run after the night calibration scripts',
+                                 formatter_class = argparse.ArgumentDefaultsHelpFormatter)
+required = parser.add_argument_group('required arguments')
+optional = parser.add_argument_group('optional arguments')
+
+required.add_argument('-r', '--run_list', help="Run number if the flat-field data",
+                      type=int, nargs="+")
+
+version,subversion=lstchain.__version__.rsplit('.post',1)
+optional.add_argument('-f', '--filters_list', help="Filter list (same order as run list)",
+                      type=int, nargs="+")
+
+optional.add_argument('-v', '--prod_version',
+                      help="Version of the production",
+                      default=f"v{version}")
+optional.add_argument('-s', '--statistics',
+                      help="Number of events for the flat-field and pedestal statistics",
+                      type=int,
+                      default=10000)
+optional.add_argument('-p', '--pedestal_run',
+                      help="Pedestal run to be used. If None, it looks for the pedestal run of the date of the FF data.",
+                      type=int)
+optional.add_argument('--time_run',
+                      help="Run for the time calibration. If None, search the last time run before or equal the first filter scan run",
+                      type=int)
+optional.add_argument('-b','--base_dir', help="Root dir for the output directory tree",type=str, default='/fefs/aswg/data/real')
+optional.add_argument('--sub_run_list',
+                      help="sub-run list to be processed.",
+                      type=int,
+                      nargs="+",default=[0])
+optional.add_argument('--sys_date',
+                      help="Date of systematics corrections (format YYYYMMDD). \n"
+                           "If '0', no corrections are applied. Default: automatically search the best date \n")
+
+
+default_config=os.path.join(os.path.dirname(__file__), "../../data/onsite_camera_calibration_param.json")
+optional.add_argument('--config', help="Config file", default=default_config)
+
+args = parser.parse_args()
+run_list = args.run_list
+filters_list = args.filters_list
+ped_run = args.pedestal_run
+prod_id = args.prod_version
+stat_events = args.statistics
+base_dir = args.base_dir
+time_run = args.time_run
+
+sub_run_list = args.sub_run_list
+config_file = args.config
+sys_date = args.sys_date
+calib_dir=f"{base_dir}/monitoring/PixelCalibration"
+
+
+def main():
+
+    if shutil.which('srun') is None:
+        print(">>> This script needs a slurm batch system. Stop")
+        return
+
+    print(f"\n--> Start reconstruct runs {run_list} and sub-runs {sub_run_list}")
+
+    # verify config file
+    if not os.path.exists(config_file):
+        raise IOError(f"Config file {config_file} does not exists. \n")
```

Comment:
Raising an exception in the main method does not really make sense. Print error and `sys.exit(2)`